packages/apollo-server-errors/src/index.ts

```diff
@@ -134,7 +134,7 @@ export function toApolloError(
 
 export interface ErrorOptions {
   code?: string;
-  errorClass?: typeof ApolloError;
+  errorClass?: new (message: string) => ApolloError;
```

Comment:
This seems fine, maybe a comment here that explains, though I also think it's pretty self-explanatory if you look at it for a second.

FWIW, I tried to improve the type here a bit with no luck - I tried to make this "condition" into an interface that `UserInputError` could implement but TS wasn't having it.

Reply:
Added a comment. FWIW I find this type to be easier to understand than `typeof ApolloError` anyway. I think the lesson for the future is to just take functions rather than class names/constructors though... easy enough to pass `(s) => new MyClass(s)`...